docs/src/main/sphinx/connector/redis.rst

```diff
@@ -138,8 +139,21 @@ This property is optional; the default is ``:``.
 References a folder within Trino deployment that holds one or more JSON
 files, which must end with ``.json`` and contain table description files.
 
+Note that the table description files will only be used by the Trino coordinator
+node.
+
 This property is optional; the default is ``etc/redis``.
 
+``redis.table-description-cache-ttl``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When adding a file end with ``.json`` to ``redis.table-description-dir`` folder,
```

Comment:
The refresh is not specific to "adding". Let's improve the sentence.

Reply:
Thank you so much for your suggestion, I updated the description.